hieroplatform/src/test/java/org/hiero/sketch/UdiBug.java

```diff
@@ -0,0 +1,38 @@
+package org.hiero.sketch;
+
+import org.hiero.sketch.table.SmallTable;
+import org.hiero.sketch.table.api.IColumn;
+import org.hiero.sketch.table.api.IMembershipSet;
+import org.hiero.sketch.table.api.IRowIterator;
+import org.hiero.sketch.table.api.ITable;
+
+import java.util.List;
+
+import static org.hiero.sketch.TableTest.SplitTable;
+import static org.hiero.sketch.TableTest.getIntTable;
+
+public class UdiBug {
```

Comment:
Since Udi has filed an issue let's call this class Issue46, since it is a reproduction for #46.